### PR TITLE
Simplify DateFilter for non two-year transaction periods

### PR DESCRIPTION
**Pull request by xtine**
Date filters that are not bound to two-year transaction periods will not have a radio selection but  simpler beginning/end input date fields. Relies on 18F/openfec-web-app#1590.

Issue: #501

screen shot 2016-09-19 at 3 04 38 pm

### CODE REVIEW

File: js/date-filter.js [hunk 1]
```diff
@@ -25,8 +25,13 @@ function DateFilter(elm) {
     oncomplete: this.validate.bind(this)
   });
 
+  if (this.$minDate.attr('data-min-date')) {
```
Comment (noahmanger) [inline js/date-filter.js]: Is this necessary? I don't think we should be setting min and max dates by default here. It's ok to have them empty.

Comment (xtine) [inline js/date-filter.js]: Is it really okay to have the date input fields empty? The page first loads with a one year period by default, so I think the fields should reflect that.

Comment (noahmanger) [inline js/date-filter.js]: It does? The nav link includes a `max_date` (which is there to hide expenditures that were incorrectly marked as happening in the future), but other than that I don't think we should be limiting by date automatically. (In principal I agree with you, though).

Comment (xtine) [inline js/date-filter.js]: Ah okay so my radio button was set to "this year" so I had mistakenly thought that was default. I've taken out default values for date input fields.

**Comment by xtine** [comment]
Date filter tags for non two-year transactional periods now removable:
screen shot 2016-09-20 at 10 25 09 am

**Comment by noahmanger** [comment]
Nice! The only thing I caught is that it doesn't seem to de-increment the filter count when removing one of these tags. (The others seem to be working as expected).

**Comment by xtine** [comment]
Had to set the non-transactional date inputs as loaded so the `handleRemoveEvent` in `filter-panel.js` decrements load count correctly.

**Pull request merged**